### PR TITLE
Update meta descriptions

### PR DESCRIPTION
**Pull request by kewitham**
This PR updates the meta descriptions for the following pages:
/topics
/resources
/topics/service-animals
/law-and-regs
/law-and-regs/title-iii-regulations
/law-and-regs/title-ii-2010-regulations
/law-and-regs/ada

### CODE REVIEW

File: _pages/resources.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 permalink: /resources/
 title: Guidance & Resource Materials
+description: "Detailed guidance documents explaining the rights and responsibilities under the ADA and the Department's regulations related to topics the ADA covers including service animals, health care and more."
```
Comment (okkays) [inline _pages/resources.md:4]: I'm not sure if it matters for these frontmatter fields, but should this not have quotes?

```suggestion
description: Detailed guidance documents explaining the rights and responsibilities under the ADA and the Department's regulations related to topics the ADA covers including service animals, health care and more.
```

File: _pages/topics.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 permalink: /topics/
 title: Featured Topics
+description: "These topic overviews are a basic starting point for understanding areas the ADA covers including service animals, parking and effective communication."
```
Comment (okkays) [inline _pages/topics.md:4]: ditto

**Review by okkays — CHANGES_REQUESTED** [review]

File: _pages/resources.md (hunk 1 above)
Comment (kewitham) [reply _pages/resources.md]: some of them do, some of them don't weirdly but I think you're right that it's not necessary

**Review by okkays — APPROVED** [review]
LGTM code-wise!

**Pull request merged**